daffodil-tdml-processor/src/main/scala/org/apache/daffodil/processor/tdml/DaffodilTDMLDFDLProcessor.scala

```diff
@@ -204,6 +202,12 @@ class DaffodilTDMLDFDLProcessor private (private var dp: api.DataProcessor)
     copy(dp = dp.withDebugger(d))
   }
 
+  override def withValidatorName(name: String): DaffodilTDMLDFDLProcessor = {
+    copy(
+      dp = dp.withValidatorName(name)
+    )
```

Comment:
Yeah, I would suggest this doesn't want to do this, but instead calls `Validator.get.make` and does `dp = dp.withValidator(...)`.